api/send_submission.php

```diff
@@ -175,7 +175,7 @@
     }
 
 
-if($is_under_21){
+if($is_under_21 && $ticket_type!='1'){
```

Comment:
Could you add spacing for readability like so: `$is_under_21 && $ticket_type != '1'`?